src/IECoreGL/bindings/TextureLoaderBinding.cpp

```diff
@@ -43,14 +43,31 @@
 
 using namespace boost::python;
 
+namespace
+{
+
+IECoreGL::TexturePtr load1( IECoreGL::TextureLoader &textureLoader, const std::string &name, int maximumResolution )
+{
+	IECore::IntDataPtr maximumResolutionData = new IECore::IntData( maximumResolution );
+	return textureLoader.load( name, maximumResolutionData.get() );
+}
+
+IECoreGL::TexturePtr load2( IECoreGL::TextureLoader &textureLoader, const std::string &name )
+{
+	return textureLoader.load( name );
+}
+
+} // namespace
+
 namespace IECoreGL
 {
 
 void bindTextureLoader()
 {
 	IECorePython::RefCountedClass<TextureLoader, IECore::RefCounted>( "TextureLoader" )
 		.def( init<const IECore::SearchPath &>() )
-		.def( "load", &TextureLoader::load )
+		.def( "load", &load1 )
+		.def( "load", &load2 )
```

Comment:
You should be able to achieve the same thing without helpers and with a single call to `def`, at the same time as providing better python help on the function signature. Something like this (untested) :

```
def( "load", &TextureLoader::load, ( arg( "fileName" ), arg( "maximumResolution" ) = numeric_limits<int>::max() ) );
```

Reply:
Thanks, much better. Addressed in 26c9dcc